cli/commands/usage.go

```diff
@@ -161,4 +161,7 @@ defaults to %s`, defaultStartTimeout.String())
 
 	logLinesUsage = fmt.Sprintf(`Count of last lines to output
 defaults to %d`, defaultLogLines)
+
+	netMsgMaxUsage = fmt.Sprintf(`Maximum count of messages the fibers handle
```

Comment:
It seems to me that this should be placed in the section with the ``pack`` usage commands

Reply:
I was wrong, pack variables are constants and we can't place this string there. Anyway, I think we have to add comment for which ``cartridge`` command this string is used